lib/net/packets.js

```diff
@@ -50,7 +49,6 @@ exports.types = {
   SENDHEADERS: 12,
   BLOCK: 13,
   TX: 14,
-  REJECT: 15,
```

Comment:
I think that if we remove the `REJECT` enum value from the `types` object, we need to shift down all the numbers for the enum values that follow to make sure that everything matches in the `typesByVal` list. I am leaning towards just leaving in `REJECT` so that it is a more simple change.

Reply:
Kinda agree with this, maybe just add a comment to the right `// deprecated` ?

Reply:
I like that solution. Will push up a fix.